docs/content/guides/7.multistore/3.patterns/5.subpath/1.subpath.md

```diff
@@ -0,0 +1,819 @@
+│   │   └── layout.tsx        # Apply store-specific class
+```
+
+#tab-2
+For Nuxt apps, update your file structure:
+
+```bash
+apps/storefront-unified-nuxt/pages/
+├── [configId]/              # Store identifier parameter
+│   ├── index.vue            # Home page
+│   ├── cart/                # Cart pages
+│   │   └── index.vue
+│   ├── checkout/            # Checkout pages
+│   │   └── index.vue
+│   ├── products/            # Product pages
+│   │   ├── index.vue
+│   │   └── [id].vue
+│   ├── some-electronics-page.vue # Electronics-specific page
+│   └── some-apparel-page.vue     # Apparel-specific page
+```
+::
+
+This structure enables routes like:
+- `/en/electronics` - Electronics store home
+- `/en/apparel` - Apparel store home
+- `/en/electronics/category` - PLP in the electronics store
+- `/en/apparel/category` - PLP in the apparel store
+
+## Updating Internal Links
+
+::warning Update Internal Links
+After setting up the path routing structure, you must update all internal links in your application to include the appropriate configId parameter. This is crucial to maintain proper navigation within each store and prevent users from accidentally switching between stores when clicking links.
+::
+
+Make sure that all your internal links and navigation components preserve the store context by including the current configId in the URLs. This applies to:
+
+- Navigation menus
+- Product links
+- Category links
+- Cart and checkout flows
+- Footer links
+- Breadcrumbs
+- Any other internal navigation elements
+
+Without this, users might unexpectedly switch between stores while browsing your site.
+
+## Store-Specific Styling
+
+There are two key steps to implement store-specific styling:
+
+### 1. Applying Store Identifier Class
+
+First, you need to apply the store identifier as a CSS class to enable different styles for each store:
+
+::tabs{:titles='["Next.js", "Nuxt"]'}
+
+#tab-1
+In your Next.js layout, apply the configId as a class:
+
+```tsx
+// apps/storefront-unified-nextjs/app/[locale]/[configId]/layout.tsx
+import { PropsWithChildren } from 'react';
+
+interface LayoutProps extends PropsWithChildren {
+  params: {
+    configId: string;
+  };
+}
+
+export default function Layout({ children, params: { configId } }: LayoutProps) {
+  return (
+    <div className={configId}>
+      {children}
+    </div>
+  );
+}
+```
+
+#tab-2
+In your Nuxt app.vue, apply the configId as a class:
+
+```vue
+// apps/storefront-unified-nuxt/app.vue
+<template>
+  <Body :class="['font-body', route.params.configId]" />
+  <NuxtLayout>
+    <NuxtPage />
+  </NuxtLayout>
+</template>
+
+<script setup lang="ts">
+const route = useRoute();
+</script>
+```
+::
+
+### 2. Defining Store-Specific CSS Variables
+
+Once the store identifier class is applied, you can define CSS variables that change based on the store:
+
+::tabs{:titles='["Next.js", "Nuxt"]'}
+
+#tab-1
+In your Next.js app, add custom CSS variables in your global styles:
```

Comment:
wasn't it already described in lines 119-200?

Reply:
ah, sorry I've resolved the conflicts incorrectly